fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1250,8 +1250,9 @@ ACTOR Future<MutationRef> writeMutation(CommitBatchContext* self,
 	static_assert(TenantInfo::INVALID_TENANT == INVALID_ENCRYPT_DOMAIN_ID);
 
 	if (self->pProxyCommitData->isEncryptionEnabled) {
-		EncryptCipherDomainId domainId = tenantId;
+		state EncryptCipherDomainId domainId = tenantId;
```

Comment:
Can we just move this to the else statement, i don't think the if uses it

Reply:
Given the `tenantId == INVALID_TEANANT_ID` is a valid thing, I updated the code to use `domainId` for encryption related things; for instance: 

```
ASSERT_NE(domainId, INVALID_ENCRYPT_DOMAIN_ID);
encryptedMutation = mutation->encrypt(self->cipherKeys, domainId, *arena, BlobCipherMetrics::TLOG); 
```

is used at the end, hence, the definition of variable is not limited to if and/or else block.

Reply:
I'm just saying to move it to line 1271 since only things below that use it

Reply:
Oh I see, yeah after your patch, it can be safely scoped to `else` block corresponding to `encryptedMutationOpt->present()`. 
Sure.